.github/workflows/build.yml

```diff
@@ -0,0 +1,110 @@
+# This pipeline builds the sfpowerscripts plugin
+
+
+name: 'sfpowerscripts Build Pipeline'
+
+on:
+  push:
+    branches:
+      - develop
+    paths-ignore:
+      - 'docs/**'
```

Comment:
For paths to ignore, we want to include README files, decision records, demoreel and prerequisites

Reply:
added the following in paths-ignore:

README files, decision records, demoreel and prerequisites